crates/lox-io/src/ndm/kvn/parser.rs

```diff
@@ -153,11 +153,23 @@ pub struct KvnValue<V, U> {
     pub unit: Option<U>,
 }
 
+#[derive(PartialEq, Debug)]
+pub enum DateOfYear {
+    DayOfMonth { month: u8, day: u8 },
+    DayOfYear { day: u8 },
```

Comment:
365 days (per year) doesn't fit in `u8`?

Reply:
Whoops.

Reply:
Fixed and added test case.